src/pages/ClaimsPage/ClaimsPage.tsx

```diff
@@ -80,9 +86,18 @@ export default function ClaimsPage() {
       </SingleColumnLayout>
     );
 
-  const claimsGroupByVault = groupBy(
-    claims.sort(c => c.id),
-    c => c.distribution.epoch.circle?.id && c.distribution.vault.vault_address
+  const claimedClaimsGroupByRow = groupBy(
+    claims.filter(c => c.txHash).sort(c => -c.id),
+    c => {
+      return claimsRowKey(c.distribution, c.txHash);
+    }
+  );
+
+  const unclaimedClaimsGroupByRow = groupBy(
+    claims.filter(c => !c.txHash).sort(c => c.id),
```

Comment:
```suggestion
    claims.filter(c => !c.txHash).sort(c => -c.id),
```

Reply:
i changed a bunch of the sort logic... 

right now I'm sorting so newest is at the top by default